src/main/resources/crafter/studio/upgrade/pipelines.yaml

```diff
@@ -326,6 +326,15 @@ pipelines:
               dest: configuration/samples/sample-urlrewrite.xml
             - src: crafter/studio/upgrade/3.1.0.24/configuration/samples/sample-webdav.xml
               dest: configuration/samples/sample-webdav.xml
+    - currentVersion: 3.1.0.24
+      nextVersion: 3.1.0.25
+      operations:
+        - type: dbScriptUpgrader
```

Comment:
Maybe it would be better to update the version as the last operation, that way if something else fails the version will not be changed.